english/quick_draw/AI_Pictionary.md

```diff
@@ -0,0 +1,5 @@
+It is a lot of fun playing pictionary with a bunch of friends. 
+
+But have you ever played pictionary with a computer, where you draw sketches and computer predicts your sketch within 20 seconds?
+
+If not, play with this awesome [AI](www.quickdraw.google.com) built by guys at Google.
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.